content/release-10-14-0/streaming-analytics-10-14-0-bundle/10_14_0.md

```diff
@@ -3,3 +3,32 @@ weight: 40
 title: Release 10.14.0
 layout: redirect
 ---
+
+### Apama correlator version
+
+This release of Cumulocity IoT Streaming Analytics includes the Apama version 10.11.3 correlator.
+See also [What's New In Apama 10.11.3](https://documentation.softwareag.com/apama/v10-11/apama10-11/apama-webhelp/index.html#page/apama-webhelp%2Fco-WhaNewInApa_10113_top.html)
+in the Apama documentation.
+
+### Improvements in Analytics Builder
+
+Samples are now provided in the model manager. These are similar to the smart rules in Cumulocity IoT. 
+The samples are intended to help you get started with creating your own models. You can view the samples, but you cannot edit or deploy them. 
+However, you can use a sample as a basis for further development by creating a new model from the sample. 
+You can then edit the new model according to your requirements and deploy it. 
+See also the information on the new **Samples tab** in [The model manager user interface](https://documentation.softwareag.com/apama/Analytics_Builder/pab10-14-0/apama-pab-webhelp/index.html#page/apamaanalyticsbuilder-webhelp%2Fco-AnaBui_the_model_manager_user_interface.html)
+in the Analytics Builder documentation.
+
+### Cumulocity IoT transport in Apama
```

Comment:
I don't think this section adds value here, especially if its about this new propertly.

Reply:
Thanks, removed.